variants/Dongle_nRF52840-pca10059-v1/variant.h

```diff
@@ -0,0 +1,184 @@
+/*
+  Copyright (c) 2014-2015 Arduino LLC.  All right reserved.
+  Copyright (c) 2016 Sandeep Mistry All right reserved.
+  Copyright (c) 2018, Adafruit Industries (adafruit.com)
+  This library is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 2.1 of the License, or (at your option) any later version.
+  This library is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef _VARIANT_NORDIC_PCA10059_
+#define _VARIANT_NORDIC_PCA10059_
+
+#define PCA10059
+
+/** Master clock frequency */
+#define VARIANT_MCK (64000000ul)
+
+#define USE_LFXO // Board uses 32khz crystal for LF
+// define USE_LFRC    // Board uses RC for LF
+
+/*----------------------------------------------------------------------------
+ *        Headers
+ *----------------------------------------------------------------------------*/
+
+#include "WVariant.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+// Number of pins defined in PinDescription array
+#define PINS_COUNT (48)
+#define NUM_DIGITAL_PINS (48)
+#define NUM_ANALOG_INPUTS (6)
+#define NUM_ANALOG_OUTPUTS (0)
+
+// LEDs
+#define PIN_LED1 (0 + 12)           // Blue LED        P1.12
+#define PIN_LED2 (0 + 6)            // Built in Green  P0.06
+
+//Green Built in LED1
+//#define PIN_LED1 (0 + 6)          // LED1            P1.15
+  
+//RGB NeoPixel LED2
+//#define PIN_LED1 (0 + 8) Red
+//#define PIN_LED1 (32 + 9) Green
+//#define PIN_LED1 (0 + 12) Blue
+
+#define LED_BUILTIN PIN_LED1
+#define LED_CONN PIN_LED2
+
+#define LED_GREEN PIN_LED1
+#define LED_BLUE PIN_LED2
+
+#define LED_STATE_ON 0              // State when LED is litted
+
+/*
+ * Buttons
+ */
+
+#ifdef PCA10059
+#define PIN_BUTTON1 (32 + 6)        // BTN_DN           P1.06 Built in button
+#endif
```

Comment:
Is the guard neccessary here? we're board specific anyway in this file.

Reply:
I will remove it